src/CPU/syscall.cpp

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
+#include <chrono>
+#include <thread>
+#include <unistd.h>
+#include <sched.h>
+#include "timer.hpp"
+#include "stats.hpp"
+#include "proc.hpp"
+
+
+using namespace std;
+
+// getpid() cache has been deprecated, see https://man7.org/linux/man-pages/man2/getpid.2.html
+
+double sys_call_time_chrono() {
+    auto start = chrono::steady_clock::now();
+    for(ssize_t i = 100000; i > 0; i--) {
```

Comment:
Can we define this as a constant?

Reply:
yeah sure, any large number works. I don't think there is any reason to change the value, so I didn't define it separately